pkg/kube/daemonset.go

```diff
@@ -37,6 +37,12 @@ func GenerateDaemonSet(instance *sfv1alpha1.SplunkForwarder) *appsv1.DaemonSet {
 			Name:  "SPLUNK_ACCEPT_LICENSE",
 			Value: licenseAccepted,
 		},
+		{	Name: "HOSTNAME",
```

Comment:
This is definitely a nit, but can this be moved down a line? Similar with line 44 so we have consistent braces
```suggestion
		{
		  Name: "HOSTNAME",
```